Rules/UseCompatibleCmdlets.cs

```diff
@@ -382,6 +444,28 @@ private bool IsValidPlatformString(string fileNameWithoutExt)
                 out os);
         }
 
+        /// <summary>
+        /// Create a string with current date/time for log file name.
+        /// </summary>
+        private string CreateLogFileName(string settingsPath)
+        {
+            string dateString = String.Format("{0:g}", DateTime.Now);
```

Comment:
This method might be better using a `StringBuilder`. The comment should specify an example of a log file name so it's clear what the logic should be doing